src/__tests__/testScreen.py

```diff
@@ -36,7 +36,7 @@
     'name': 'selectTwoCommandMode',
     'input': 'absoluteGitDiff.txt',
     'inputs': ['f', 'j', 'f', 'c'],
-    'pastScreen': 1
+    'pastScreen': 3
```

Comment:
Yeah -  because we now take the screenshot when you "refresh". Before we only did it on erase and we don't always call erase on the entire screen only when we clear the entire screen. When we dirty a line (say from up down or j f) we just call refresh.